lib/workers/repository/extract/index.ts

```diff
@@ -48,6 +48,7 @@ export async function extractAllDependencies(
   const extractResults = await Promise.all(
     extractList.map(async (managerConfig) => {
       const packageFiles = await getManagerPackageFiles(managerConfig);
+      //TODO: updateInternalDeps can be removed from WorkerExtractConfig once we move this
```

Comment:
not sure why 🤔

Reply:
If this option changes then deps property will change, which means that our `ExtractResult` is different from last time.

Reply:
i think this is no longer true with #15790, as `updateInternalDeps` is no longer used in `getManagerPackageFiles`. It's used a few lines below instead: `config.updateInternalDeps`

Reply:
It is out of the manager logic which is true.
Currently the decision is being made in extract section, but we need to move it to the lookup section where it actually is being used.
 [Need to move here](https://github.com/renovatebot/renovate/blob/5c803f2e414ac84300f037ffa6bb33dc3f558ee6/lib/workers/repository/process/fetch.ts#L29)

Reply:
Why it's still there, when it's no longer used? 🤔 lets remove it in a seperate PR and merge before this, so we don't need this nasty todo